index.bs

```diff
@@ -894,8 +894,8 @@ enum MLOperandType {
 };
 
 dictionary MLOperandDescriptor {
-  // The operand type.
-  required MLOperandType type;
+  // The operand data type.
+  required MLOperandDataType dataType;
```

Comment:
This change would break trial for WebNN samples/WebNN EP on Chrome/Edge stable for a period of time (at lease one round of Stable release cycle), we can do workaround by adding a duplicated entry to the JS code, e.g. 
`const aTensorDesc = {type: 'float32', dataType: 'float32', dimensions: [1, 3, 3, 3]};`

And we may encounter more such kind of API breaking before the spec becoming mature. It's painful for developers who want to try WebNN early.

Reply:
Yeah, that's why I proposed this right after the recent big spec refactor 446, but before v2 ops were added, because up until this time, we've liberally updated the spec (e.g. changing signed dimension to unsigned, MLOperator -> MLActivation...), but it will become harder over time. 🕗...

Reply:
I think this is a good discussion about how much we should tolerate a breaking change at this stage in the spec life cycle, now that folks started to implement it. And although this probably won't be the last breaking change before the spec enters a more stable stage (still to be defined when), the bar for a breaking change to go in should continue to rise overtime. 

So, name change is a breaking change, but if it's early enough, we can still look away. But I don't think this period should last long.